cypress/cypress/component/Input/BasicInputField.cy.tsx

```diff
@@ -52,6 +52,30 @@ describe("Input.cy.tsx", () => {
     cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: randomNumber });
   });
 
+  it("number accepts step", () => {
+    const name = faker.random.alpha(10);
+    const schema = yup.object().shape({
+      [name]: yup.number(),
+    });
+
+    const randomNumber = faker.datatype.number();
+    const step = 2;
+
+    cy.mount(
+      <Form onSubmit={cy.spy().as("onSubmitSpy")} resolver={yupResolver(schema)}>
+        <Input type="number" name={name} label={name} step={step} />
+
+        <input type={"submit"} />
+      </Form>,
+    );
+
+    cy.contains("label", name).click().type(randomNumber.toString());
+    cy.get(`input[id=${name}]`).should("have.attr", "step", step.toString());
+    cy.get("input[type=submit]").click({ force: true });
+
+    cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: randomNumber });
```

Comment:
@giorgioverzicco I think you are not testing nothing. You should test that, clicking on the increase button, the result is randomNumber + step

Reply:
@manni497 now we have an expected result to check when an uparrow is triggered on the input the result must be equal